org-formation/600-access/_tasks.yaml

```diff
@@ -173,6 +173,30 @@ DnTDevCIServiceAccounts:
       - !Ref DnTDevAccount
     Region: us-east-1
 
+CostExplorerAccessRole:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/IAM/managed-policy.yaml
+  StackName: cost-explorer-access-role
+  DefaultOrganizationBinding:
+    Account: !Ref MasterAccount
+    Region: !Ref primaryRegion
+  Parameters:
+    PolicyDocument:
+      Version: 2012-10-17
+      Statement:
+        - Sid: CostExplorerAccess
+          Effect: Allow
+          Action:
+            - aws-portal:ViewBilling
+            - aws-portal:ViewUsage
```

Comment:
do you think we can replace these with the AWS managed policy `AWSBillingReadOnlyAccess`?

Reply:
according to https://aws.permissions.cloud/managedpolicies/AWSBillingReadOnlyAccess it only grants `aws-portal:ViewBilling`, so I don't think it's very useful; but I'm also surprised that that isn't mentioned anywhere under https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/security-iam.html